CONTRIBUTING.md

```diff
@@ -68,4 +86,4 @@ you can use one of some generic reasons like "Improve documentation.",
 
 The project now enables [DCO check](https://github.com/probot/dco#how-it-works) and the commit message must contain a `Signed-off-by` line for [Developer Certificate of Origin](https://developercertificate.org/)
 
-You can use option `-s` for `git commit` to automatically add a `Signed-off-by` to the commit message. 
\ No newline at end of file
+You can use option `-s` for `git commit` to automatically add a `Signed-off-by` to the commit message. 
```

Comment:
Why is this line marked as a change by GitHub.. Did you do anything to it?

Reply:
It's the last line of a file, and typically this has a blank newline after it (eg `blah blah\n` is the raw content of a 1 line file). My Git config is is set to automatically add this to any file without it.

Notably, this is part of the POISIX standard: http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206, so there isn't a harm to following this.